java/java.lsp.server/vscode/README.md

```diff
@@ -21,80 +21,39 @@
 
 -->
 
+This is a technology preview of [Apache NetBeans](http://netbeans.org)
+based extension for VSCode. Use it to get all the _goodies of NetBeans_
```

Comment:
Please use VS Code (include space), which is the official abbreviation used by Microsoft.

Reply:
Done in a69fb9e